### PR TITLE
Merge 19.2 code freeze with conflicts resolution

### PR DESCRIPTION
**Pull request by mokagio**
Supersedes #17899. I got conflicts on `Podfile` and `Podfile.lock` because the WordPressKit beta version changed in the time it took me to open the original PR. WordPress iOS moves fast 😄

### CODE REVIEW

**Review by mokagio — COMMENTED** [review]
- [x] Switch to production Pods
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] `Localizable.strings` updated 
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] Version update in `.xcconfig` and `download_metadata.swift`

File: WordPress/Jetpack/Resources/release_notes.txt [hunk 1]
```diff
@@ -1,18 +1,8 @@
...
-- Weekly roundup notifications are now showing up, making it easier to stay on top of the latest news.
-- When you connect your Facebook account to Publicize, you’ll now be logged in successfully.
-- Images with a copyright symbol in the file name will now upload or appear in the Share Sheet properly.
+* [*] Block editor: Replacing the media for an image set as featured prompts to update the featured image [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3930]
+* [***] Block editor: Font size and line-height support for text-based blocks used in block-based themes [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4519]
+* [**] Some of the screens of the app has a new, fresh and more modern visual, including the initial one: My Site. [#17812]
+* [**] Notifications: added a button to mark all notifications in the selected filter as read. [#17840]
+* [**] People: you can now manage Email Followers on the People section! [#17854]
+* [*] Stats: fix navigation between Stats tab. [#17856]
+* [**] Threaded comments: comments can now be moderated via a drop-down menu on each comment. [#17888]
```
Comment (ParaskP7) [inline WordPress/Jetpack/Resources/release_notes.txt:7]: 👋 @leandroalonso !

Can you please make sure that `Threaded comments` are applicable to `Jetpack` as well? 🙏 

PS: I think it does, just wanted to double check with you.

**Review by ParaskP7 — APPROVED** [review]
LGTM! 🎉

**Comment by mokagio** [comment]
At the time of writing this, the Installable Builds tasks have not yet run.

image

Looking at the pipeline that [run on Buildkite](https://buildkite.com/automattic/wordpress-ios/builds/4982), the steps weren't even added to it.

The reason, I'm pretty sure, is that when Buildkite run, I hadn't open this PR yet. You can see that the environment for the build shows no pull request:

image

I'm now going to restart the build to see if Buildkite picks up that there now is a PR associated with the commit 🤞 

image

**Comment by mokagio** [comment]
> I'm now going to restart the build to see if Buildkite picks up that there now is a PR associated with the commit 🤞

The answer is no 😞 

Direct link to the build: https://buildkite.com/automattic/wordpress-ios/builds/4991#558b0e4c-9367-4943-98de-b0825cf0bf22

**Comment by mokagio** [comment]
> > I'm now going to restart the build to see if Buildkite picks up that there now is a PR associated with the commit 🤞
>
> The answer is no 😞

`trunk` moved on and we now have conflicts. This means I'll have to push a new commit and the task will kick off 😄 

image

**Comment by jkmassel** [comment]
You can test the <strong>Jetpack</strong> changes on this Pull Request by downloading it from AppCenter <a href='https://install.appcenter.ms/orgs/automattic/apps/jetpack-installable-builds/'>here</a> with build number: <code>pr17900-fe79495</code>. IPA is available <a href='https://appcenter-filemanagement-distrib3ede6f06e.azureedge.net/4c63b88d-a600-4503-884c-641aa51c1df9/Jetpack%20Alpha.ipa?sv=2019-02-02&sr=c&sig=u5FYAauyRRTU8YjYpKk2%2FujydjpGmRySwzrvJ9OVlBM%3D&se=2022-02-09T11%3A36%3A57Z&sp=r'>here</a>. If you need access to this, you can ask a maintainer to add you.

**Comment by jkmassel** [comment]
You can test the <strong>WordPress</strong> changes on this Pull Request by downloading it from AppCenter <a href='https://install.appcenter.ms/orgs/automattic/apps/WPiOS-One-Offs/'>here</a> with build number: <code>pr17900-fe79495</code>. IPA is available <a href='https://appcenter-filemanagement-distrib2ede6f06e.azureedge.net/2c5fde6a-99af-4f13-9bd9-eaacf688edb2/WordPress%20Alpha.ipa?sv=2019-02-02&sr=c&sig=q%2BkuGU47YpbTAG%2BzG4Av9QnHhMvAMZbZZtjz4e2uH8E%3D&se=2022-02-09T11%3A39%3A08Z&sp=r'>here</a>. If you need access to this, you can ask a maintainer to add you.

File: WordPress/Jetpack/Resources/release_notes.txt (hunk 1 above)
Comment (leandroalonso) [reply WordPress/Jetpack/Resources/release_notes.txt:7]: Yep, it is! (cc @ScoutHarris)

**Pull request merged**